### PR TITLE
zh-cn: init translation of location protocol property

### PR DESCRIPTION
**Pull request by zjffun**
Original: https://github.com/mdn/content/blob/eb99a4597802f746275e0d61b31cd2f42b398eab/files/en-us/web/api/location/protocol/index.md

Detail: https://dochub.zjffun.com/translate/mdn-zh-cn/mdn/translated-content/main/files/zh-cn/web/api/location/protocol/index.md

### CODE REVIEW

File: files/zh-cn/web/api/location/protocol/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "location: protocol property"
```
Comment (yin1999) [inline files/zh-cn/web/api/location/protocol/index.md:2]:
```suggestion
title: location：protocol 属性
```

File: files/zh-cn/web/api/location/protocol/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{{domxref("Location")}} 接口的 **`protocol`** 属性是一个表示 URL 协议方案的字符串，包含结尾的 `':'`。
+
+## 值
+
+一个字符串。
+
+## 示例
+
+```js
+// 让我们在文档中加入一个 <a id="myAnchor" href="https://developer.mozilla.org/zh-CN/Location.protocol"> 元素
```
Comment (yin1999) [inline files/zh-cn/web/api/location/protocol/index.md:17]:
```suggestion
// 在文档中加入一个 <a id="myAnchor" href="https://developer.mozilla.org/zh-CN/Location.protocol"> 元素
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

**Comment by zjffun** [comment]
@yin1999 Thanks. Your translations are better.

**Review by yin1999 — APPROVED** [review]
👍

**Pull request merged**